daemon/src/monitor.rs

```diff
@@ -666,8 +665,8 @@ impl<C> xtra::Handler<Sync> for Actor<C>
 where
     C: bdk::electrum_client::ElectrumApi + Send + 'static,
 {
-    async fn handle(&mut self, _: Sync, _ctx: &mut xtra::Context<Self>) {
-        log_error!(self.sync());
+    async fn handle(&mut self, _: Sync, _ctx: &mut xtra::Context<Self>) -> Result<()> {
```

Comment:
Okay, I think I get you. The compiler doesn't complain, but I'll revert this change and log in the handler like we used to.

I think it wouldn't actually kill the notification loop since it ignores the variant of the inner result:

```rust
async move {
    Delay::new(duration).await;
    while addr.do_send(constructor()).is_ok() {
        Delay::new(duration).await
    }
};
```

But we would lose the logging of the error nonetheless.